### ISSUE TITLE
Terminus Build Env Create Takes too long

### ISSUE THREAD

**Issue by ataylorme** [pantheon-systems/terminus-build-tools-plugin#244]
From https://github.com/pantheon-systems/terminus/issues/1995

### Expected behavior
terminusbuild:env:create take 3.5 to 4 minutes to run. We run in CI environment and this takes too long and when db size is bigger it will only increase. I would expect it take a minute or so to spin up Multidev.

### Actual behavior
It takes 3.5 to 4 minutes for terminus build:env:create

### Steps to reproduce the behavior
Using Terminus 2.0.1
1. Run terminus:build:env:create in CI environment 
2. Twiddle your thumbs with the rest of your team while you wait for this command to run.

**Comment by ataylorme** [comment]
@msauberPrincipal are you using the `--clone-content` option with `build:env:create`? There was some discussion in #206 around DB clone times.

For [my personal site](https://github.com/ataylorme/ataylorme-wordpress) I have adopted a strategy of creating the multidev earlier in the CI pipeline, running it concurrently with code sniffing, unit testing and building PHP assets.

Once all that is done, another job runs to deploy to Pantheon using `build:env:push`. Since the multidev was already created previously, pushing the code to Pantheon takes less time.

The multidev creation still takes time but since it runs in parallel with other jobs the overall build time decreases.

**Comment by ataylorme** [comment]
Closing this issue due to inactivity

**Comment by pattonwebz** [comment]
I wonder if this issue could be re-opened/revisited?

In the project I am working on we are seeing 8 minutes multidev creation time and it's by far the slowest part of our build. We are running the build command to create the multidev in parallel but the deploy job still has to wait on the multidev creation.

Our database is not _huge_ and we have already tried running a build excluding file copy and it still takes a very long time.

**Comment by sugaroverflow** [comment]
@pattonwebz if you'd like to, please open a new issue and reference this one.
